extensions/wikia/MyHome/templates/feed.tmpl.php

```diff
@@ -10,7 +10,11 @@
 		<?php } else { ?>
 			<span class="title"><?php print htmlspecialchars($row['title']) ?></span>
 <?php } ?>
-			<cite><?php print FeedRenderer::getActionLabel($row); ?><?php print ActivityFeedRenderer::formatTimestamp($row['timestamp']); ?><?php print FeedRenderer::getDiffLink($row); ?></cite>
+			<?php
+				$timestamp = ActivityFeedRenderer::formatTimestamp( $row['timestamp'] );
+				$timestamp = $timestamp ? ' • ' . $timestamp : '';
+			?>
+			<cite><?php print $timestamp; ?><?php print FeedRenderer::getActionLabel($row); ?><?php print FeedRenderer::getDiffLink($row); ?></cite>
```

Comment:
Shouldn't we move it to i18n to allow translators make decision about the order of these labels (timestamp / action label)?